src/main.rs

```diff
@@ -43,13 +47,24 @@ fn main() -> Result<(), Box<dyn Error>> {
 }
 
 fn print_total(total: usize) {
+    #[cfg(windows)]
+    colored::control::set_virtual_terminal(true).ok();
```

Comment:
Why exactly here?

Maybe it would be more correct to make a separate function(with `сfg(windows)`) and call in the body of `main`?
So that we can be sure that the add-in is applied and does not depend on order function calls?

Reply:
Agree 👍